lib/generate-page-element.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+'use strict'
+
+const { v4: uuid } = require('uuid')
+const { validateWithPageElementSchema } = require('../lib/forms-validation.js')
+const generateFormElement = require('./generate-form-element')
+
+function generatePageElement(
+  formElementGenerationData /* : mixed */,
+) /* :PageElement */ {
+  if (!formElementGenerationData) {
+    throw new Error(
+      'no page element generation data provided, please provide this data to generate a page element',
+    )
+  }
+  const formElement /* : Object */ = {
+    ...formElementGenerationData,
+    type: 'page',
+    id: uuid(),
```

Comment:
I think we want to allow them to have their own `id` if they want.

Reply:
I agree but the generateFormElement also sets the id

Reply:
Happy to change both though since its not really a breaking change